packages/compose/src/hooks/use-throttle/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { throttle } from 'lodash';
+import { useMemoOne } from 'use-memo-one';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Throttles a function with Lodash's `throttle`. A new throttled function will
+ * be returned and any scheduled calls cancelled if any of the arguments change,
+ * including the function to throttle, so please wrap functions created on
+ * render in components in `useCallback`.
+ *
+ * @param {...any} args Arguments passed to Lodash's `throttle`.
+ *
+ * @return {Function} Throttled function.
+ */
+export default function useThrottle( ...args ) {
```

Comment:
Why did we add this hook? it seems unused?

Reply:
It's used in the provider

Reply:
ahhh I missed it because Github kept that one closed :P Sorry